core/src/elements/ons-back-button.js

```diff
@@ -74,15 +74,17 @@ class BackButtonElement extends BaseElement {
    *  [ja]バックボタンの見た目を指定します。[/ja]
    */
 
-  createdCallback() {
-    contentReady(this, () => {
-      if (!this.hasAttribute('_compiled')) {
-        this._compile();
-      }
+  constructor(self) {
+    self = super(self);
+
+    contentReady(self, () => {
+      self._compile();
```

Comment:
We can do this instead: 
https://github.com/WebReflection/document-register-element#skipping-the-caveat-through-extends

So we use `init` method instead of `constructor` but we can use `this`.

Which do you prefer? I actually prefer the using `init` method since we need less code and can use `this` variable.